.github/cargo_local_patch.py

```diff
@@ -71,14 +79,16 @@ def process_cargo_file(cargo_toml_path, dep_path_mapping, start_directory):
     with open(cargo_toml_path, 'w') as file:
         toml.dump(data, file)
 
-def main(directory, dep_path_mapping):
-    cargo_toml_files = find_cargo_toml_files(directory)
+def main(directory, dep_path_mapping, ignore_dirs):
+    cargo_toml_files = find_cargo_toml_files(directory, ignore_dirs)
     for cargo_toml_path in cargo_toml_files:
         process_cargo_file(cargo_toml_path, dep_path_mapping, directory)
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Modify dependencies in Cargo.toml files within a directory.")
     parser.add_argument("directory", help="Directory to search for Cargo.toml files")
+    parser.add_argument("--ignore", nargs="+", default=[],
+                        help="List of directories to ignore for patching")
```

Comment:
You should remove the `nargs="+"` I think, without it you are able to use the command like this:
`cargo_local_patch.py --ignore lib --ignore lib2 .` which I think makes more sense

Reply:
sorry and also add `action="append"`